src/Parser/FrostMage/Modules/Items/ZannesuJourney.js

```diff
@@ -0,0 +1,45 @@
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import getDamageBonus from '../MageCore/GetDamageBonus';
+
+const DAMAGE_BONUS = .35;
+
+class ZannesuJourney extends Module {
+
+  static dependencies = {
+		combatants: Combatants,
+	};
+
+  damage = 0;
+  stackCount = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasWaist(ITEMS.ZANNESU_JOURNEY.id);
+  }
+
+  on_byPlayer_cast(event) {
+    if (event.ability.guid !== SPELLS.BLIZZARD.id) {
+      return;
+    }
+    const buff = this.combatants.selected.getBuff(SPELLS.ZANNESU_JOURNEY_BUFF.id,event.timestamp);
+    this.stackCount = buff && buff.stacks || 0;
```

Comment:
You're right, this should be `this.stackCount = buff && (buff.stacks || 0);`
The lack of parens makes the order of operations unclear.

Reply:
Out of curiosity, what would happen to the getDamageBonus() if the stack count is 0 or undefined? Would it register that Blizzard as doing 0 total damage? or would it register it as doing the normal damage with nothing extra?

Also it seems that each of these output differently when there is no stacks. Not sure which is right

(buff && buff.stacks) || 0
this Outputs a 0 when there is 0 stacks

buff && (buff.stacks || 0)
This Outputs "undefined" when there is 0 stacks ... but it doesnt error

Reply:
Oh oops, yeah I made the wrong suggestion, should be `(buff && buff.stacks) || 0`

This is using javascript evaluating undefined as false when coerced to be a boolean. Basically that line says "if buff is defined and buff.stacks is defined, set to buff.stacks, otherwise set to 0".